### ISSUE TITLE
Onnx model export fail

### ISSUE THREAD

**Issue by tkuanlun350** [ultralytics/yolov5#3838]
## 🐛 Bug
using export.py convert pretrained yolov5s.pt to onnx fail without error message

## To Reproduce (REQUIRED)
git clone
pip install -r requirements.txt
pip install coremltools>=4.1 onnx>=1.9.0 scikit-learn==0.19.2
[download yolov5s.pt and save into models folder]
python export.py --weights models/yolov5s.pt --img 640 --batch 1

message showed until program shutdown

Fusing layers...
Model Summary: 224 layers, 7266973 parameters, 0 gradients

PyTorch: starting from models/yolov5s.pt (14.8 MB)
TorchScript: starting export with torch 1.7.1...
C:\Research\detection\yolov5\models\yolo.py:58: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if self.grid[i].shape[2:4] != x[i].shape[2:4] or self.onnx_dynamic:
TorchScript: export success, saved as models/yolov5s.torchscript.pt (29.4 MB)
ONNX: starting export with onnx 1.9.0...

## Expected behavior
an onnx model output 

## Environment
Windows

**Comment by glenn-jocher** [comment]
@tkuanlun350 ONNX export works correctly on windows per our latest CI check one hour ago:
https://github.com/ultralytics/yolov5/runs/2951098934?check_suite_focus=true
<img width="1551" alt="Screenshot 2021-06-30 at 13 24 24" src="https://user-images.githubusercontent.com/26833433/123952604-812de800-d9a6-11eb-982f-64341c951ce6.png">


iIt appears you may have environment problems. Please ensure you meet all dependency requirements if you are attempting to run YOLOv5 locally.  If in doubt, create a new virtual Python 3.8 environment, clone the latest repo (**code changes daily**), and `pip install -r requirements.txt` again. We also highly recommend using one of our verified environments below. 


## Requirements

**Python 3.8** or later with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) dependencies installed, including **`torch>=1.7`**. To install run:
```bash
$ pip install -r requirements.txt
```


## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab and Kaggle** notebooks with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>


## Status

![CI CPU testing](https://github.com/ultralytics/yolov5/workflows/CI%20CPU%20testing/badge.svg)

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are passing. These tests evaluate proper operation of basic YOLOv5 functionality, including training ([train.py](https://github.com/ultralytics/yolov5/blob/master/train.py)), testing ([test.py](https://github.com/ultralytics/yolov5/blob/master/test.py)), inference ([detect.py](https://github.com/ultralytics/yolov5/blob/master/detect.py)) and export ([export.py](https://github.com/ultralytics/yolov5/blob/master/models/export.py)) on MacOS, Windows, and Ubuntu.

**Comment by tkuanlun350** [comment]
Solved by install new environment with python=3.8
